tensilelite/Tensile/SolutionStructs.py

```diff
@@ -2461,9 +2470,16 @@ def assignDerivedParameters(state):
         state["UnrollMajorLDSA"] = 1
         state["UnrollMajorLDSB"] = 1
     else: # mac instruction
-      state["TransposeLDS"] =  0
-      state["UnrollMajorLDSA"] = False
-      state["UnrollMajorLDSB"] = False
+      if state["UseDotInstruction"]:
+        # dot2: force TLDS = 2
```

Comment:
Should support -1
      if state["TransposeLDS"] == -1:
            state["TransposeLDS"] = 2